uw-frame-components/portal/widgets/services.js

```diff
@@ -19,11 +19,20 @@ define(['angular'], function(angular) {
           }
         })
         .catch(function(error) {
+          return getErrorPage(fname);
```

Comment:
Does this `return` placement before the two following `$log.` statements mean those log statements are never called?